### PR TITLE
EREGCSC-2419 Documentation for processes

### PR DESCRIPTION
**Pull request by thwalker6**
Resolves # EREGCSC-2419

**Description-**

Adding some documentation to the readmes over various things.  How our regulations full text search works, information regarding serverless functions, a little to the text extractor.

**This pull request changes...**

- Adds documentation
- Removed make python.unittest(does not work, use test.pytest instead)

**Steps to manually verify this change...**

1. Read it make sure it sounds ok to you.

### CODE REVIEW

**Review by peggles2 — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -116,6 +116,7 @@ In order to update your local data with the most recent version of production yo
 If adding a new model to this process please refer to the adding a new model process below.
 
 1.  Connect to the VPN and run the following command below for postgresql in the command line.  Replace db_address and port_number with the database address and port number respectively.  A file called `backup.sql` should populate in the directory the command is run in.
+  - Django bases the tables off of ApplicationName_model.  So whenever resources_* will pul in all resourcces tables from postgres.  If you need to add a new model just follow the naming convention.
 ```
```
[the remark this answers is not in the document]
Comment (cgodwin1) [reply README.md:120]: I think the naming convention is referring to "ApplicationName_Model" from before.

[the remark this answers is not in the document]
Comment (thwalker6) [reply README.md:120]: Caleb is correct.  If you look at the tables in sql that is the naming convention of the tables.  Its all lowercase letters for the file names.

**Review by peggles2 — COMMENTED** [review]
LGTM

**Comment by thwalker6** [comment]
> LGTM

You didnt actually approve it :D

**Review by cgodwin1 — APPROVED** [review]

**Review by peggles2 — APPROVED** [review]

**Pull request merged**